apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -96,6 +101,7 @@ export type FormValues = {
   hosts: { userId: number; isFixed: boolean }[];
   bookingFields: z.infer<typeof eventTypeBookingFields>;
   availability?: AvailabilityOption;
+  bookerLayouts: BookerLayoutSettings;
```

Comment:
This event type page only does the logic to get and pass the form properties into the mutation. And it does the validation here too. See methods down below.